src/components/ChatLog.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import './ChatLog.css';
+import ChatEntry from './ChatEntry';
+import PropTypes from 'prop-types';
+
+const ChatLog = ({ entries, updateChatEntry }) => {
+return (
+    <ul className="chat-entry local">
+    {entries.map((message) => {
```

Comment:
👍 Nice use of `map` to convert from our array of message data to an array of `ChatEntry` components.